crypto/include/internal/evp_int.h

```diff
@@ -44,7 +44,7 @@ struct evp_pkey_method_st {
     int pkey_id;
     int flags;
     int (*init) (EVP_PKEY_CTX *ctx);
-    int (*copy) (EVP_PKEY_CTX *dst, EVP_PKEY_CTX *src);
+    int (*dup) (EVP_PKEY_CTX *dst, EVP_PKEY_CTX *src);
```

Comment:
I don't think this is a good idea unless you want to break the API.

Reply:
This is an internal function, I just change the name of the field in that internal structure to say what the functions actually do.